Monet_Harkin/EmailValidate/server.py

```diff
@@ -0,0 +1,55 @@
+import re
+from flask import Flask, render_template, request, redirect, flash
+from mysqlconnection import MySQLConnector
+
+app = Flask (__name__)
+mysql = MySQLConnector(app, "email_python_test")
+app.secret_key = "email"
+
+@app.route("/")
+def index():
+	query = "SELECT * FROM users"
+	emails = mysql.query_db(query)
+	return render_template("index.html", all_emails=emails)
```

Comment:
Since you're actually not using any email data on your index.html page, the query isn't needed here